src/components/HouseJoin/index.js

```diff
@@ -2,21 +2,47 @@ import React, { useState } from "react";
 import { useToasts } from "react-toast-notifications";
 import { Input } from "antd";
 
+import HTTPClient from "../../HTTPClient";
+
 import "./index.css";
 
+const client = new HTTPClient(process.env.REACT_APP_API_URL, {
+  Authorization: localStorage.getItem("token"),
+});
+
 const { Search } = Input;
 
-const HouseJoin = () => {
+const HouseJoin = ({ location }) => {
```

Comment:
The JWT returned is *really* long. I think it's better to just allow the user to send the link – i.e. just tell them "this is the join code – send it to someone to allow them to join your house".